VERSIONING.md

```diff
@@ -0,0 +1,63 @@
+# Versioning strategy
+
+This repository follows a [CalVer](https://calver.org/)-type approach to versioning, in contrast to the majority of
+other PyGranta and PyAnsys projects which follow [SemVer](https://semver.org/). 
+
+This package is versioned according to the following scheme: `YYYY.[1,2].MINOR`
+
+The reasons for this are described below.
+
+# Advantages of this approach
+
+This repository only contains code generated from OpenAPI JSON definition files, which themselves are versioned 
+with Granta MI. Granta MI follows a CalVer-type approach of `YYYY.[1,2]`, with two versions released per year.
+
+These OpenAPI JSON definition files are _backwards-compatible_, but not _forwards-compatible_. In other words,
+the API definition from an older version of Granta MI is compatible with a newer version, but not the other way
+round.
+
+Given that multiple versions of Granta MI are often used in parallel for a number of releases, a key requirement
+of the `ansys-grantami-serverapi-openapi` package and other PyGranta packages is that users can quickly and easily
+identify the correct package version for a specific version of Granta MI.
+
+To simplify this relationship between Granta MI version and `ansys-grantami-serverapi-openapi` version, we use the
+Granta MI version number as the first two parts of the version number for releases from this repository. For example:
+
+* Granta MI 2023 R2 -> `ansys-grantami-serverapi-openapi` `v2023.2.0`, `v2023.2.1`, etc.
+* Granta MI 2024 R1 -> `ansys-grantami-serverapi-openapi` `v2024.1.0`, `v2024.1.1`, etc.
+
+As a result, to install a version that is guaranteed to work with your version of Granta MI, just specify:
+
+`pip install ansys-grantami-serverapi-openapi ~= 2023.1.0`
+
+# Potential alternatives
```

Comment:
'Potential alternatives' sounds like we might possibly change to these in the future.

If the decision has definitely been made to go with CalVer, and this section is just to explain why we are not doing "X other thing", then we should say so explicitly - e.g. "Alternative strategies that were considered"